### ISSUE TITLE
[Feature] Ability for loading/unloading graphs to/from disk

### ISSUE THREAD

**Issue by JoonasC** [FalkorDB/FalkorDB#13]
I understand that this feature might not be possible to  implement if Falkor remains tied to Redis as a module, nevertheless, one of the problems I had with RedisGraph was that at any given time I would have a set of graphs that I would be actively querying, while the graphs outside of that set would be wasting the limited amount of RAM on my VPS. It would be nice if Falkor had a command or the ability to automatically load/unload graphs to/from disk in order to remediate this issue.

**Comment by swilly22** [comment]
Well, I believe you can achieve this with the following sequence:
Assuming you want to unload graph `G`

Dump graph `G` to disk:
```sh
redis-cli -D "" --raw dump G > G.dump
```

Delete G from key-space using Redis `DEL` command

Restore graph `G` from disk whenever you need it:
```sh
redis-cli -X dump_tag restore G 0 dump_tag replace < G.dump
```

**Comment by JoonasC** [comment]
True, that could work. It would still be nice if this sort of functionality couls be implemented into Falkor itself, perhaps using an LRU eviction strategy.

**Comment by swilly22** [comment]
I see, assuming we're low on memory, evict "cold" graphs to disk, reload required missing graphs automatically.

**Comment by JoonasC** [comment]
Maybe the "enhancement" tag could be added since this is not a question at this point?
